projects/tuist/fixtures/ios_app_with_watch_application/WatchApp/Tests/WatchTests.swift

```diff
@@ -0,0 +1,9 @@
+import XCTest
+
```

Comment:
nit: we can do something like

```suggestion
@testable import WatchApp
```

And attempt to access a type defined within in the test for completeness

Reply:
Good point, added